Added Preset effect i made a pull request for for the agot submot, some caveats in the description

lol3003 opened the pull request:
added preset to agot file of presets     -for education Button effects     -special agot         -education_martial_prowess_5             -tier 5 is not in vanilla but in agot

Not added (future to do)
    -special agot traits of heir
    -special agot trait levels in general

lol3003 committed:
Update DI_present_effects.txt

added preset to agot file of presets     -for education Button effects     -spiecal agot         -education_martial_prowess_5             -tier 5 is not in vanilla but in agot

Not added (future to do)
    -special agot traits of heir
    -spcial agot trait levels in general

Lithane97 commented on common/scripted_effects/DI_present_effects.txt:
```diff
@@ -861,25 +861,145 @@ remove_health_conflict_forbadgene_effect = {
     if = { limit = { has_trait = education_learning_2 } remove_trait = education_learning_2 }
     if = { limit = { has_trait = education_learning_3 } remove_trait = education_learning_3 }
     if = { limit = { has_trait = education_learning_4 } remove_trait = education_learning_4 }
+    if = { limit = { has_trait = education_learning_4 } remove_trait = education_learning_5 }
```
There's a typo here, education_learning_4 should be education_learning_5

lol3003 replied:
Thanks, fixed

Lithane97 commented on common/scripted_effects/DI_present_effects.txt:
```diff
@@ -861,25 +861,145 @@ remove_health_conflict_forbadgene_effect = {
+    if = { limit = { has_trait = education_martial_prowess_3 } remove_trait = education_martial_prowess_4 add_trait = education_martial_prowess_5 }
+    
+    if = { limit = { has_trait = lifestyle_blademaster } add_trait_xp = { trait = lifestyle_blademaster value = 100 } }
+    if = { limit = { has_trait = lifestyle_mystic } add_trait_xp = { trait = lifestyle_herbalist value = 100 } }
```
Checking for trait mystic, but applying xp to trait herbalist. Should be adding xp to trait lifestyle_mystic

lol3003 replied:
Thanks, Fixed

lol3003 committed:
Fixed issues

Fixed the wrong check for removing existing education
Fixed add auxiliary traits under boost education should now boost mystic correctly

lol3003 commented:
Right now, if you play against the Boost education button, it is there, but it does nothing without this change. 
But it would be confusing to update on Steam now, since people are looking for updates on the DLC, so maybe leave it as is, on Steam, for now.

ironxandrea-png commented:
hello, is it possible to also get an upload for the latest version of this particular agot patch here? thanks!
